certified-connectors/Jedox OData Hub/readme.md

```diff
@@ -0,0 +1,44 @@
+
+## Jedox OData Hub Connector
+The Jedox OData Hub exposes functionality from different Jedox components into a powerful REST Api using the OData V4 standard.
+
```

Comment:
@NGromann The publisher section is required. Please find a template for `readme.md` [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates)